tools/wasm/air-near-contract/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "air-near-contract"
-version = "0.43.1"
+version = "0.1.0"
```

Comment:
It's version should match `air`'s version, as it is just a thin wrapper.

Reply:
makes sense